### PR TITLE
feat: auth page structure

### PR DESCRIPTION
**Pull request by banahaker**
# Description

I don't know wtf is our file structure for auth pages such as `BlockLoginPage`, `BlockSignUpPage` etc.
Obviously, those components is a PAGE not a BLOCK, so I change the structure of that.

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/router/index.ts [hunk 1]
```diff
@@ -9,51 +9,51 @@ const routes: RouteRecordRaw[] = [
...
     path: '/account',
     name: 'Account',
-    component: () => import('@/views/RouterView.vue'),
+    component: () => import('@/pages/RouterView.vue'),
     children: [
       {
         path: 'login',
         name: 'Login',
-        component: () => import('@/components/Auth/BlockLoginPage.vue'),
+        component: () => import('@/pages/Home/LoginPage.vue'),
```
Comment (SiongSng) [inline src/router/index.ts]: 話說這個東西不是在 `Account` 底下嗎？
怎麼會是用 `Home`

**Review by SiongSng — CHANGES_REQUESTED** [review]

File: src/router/index.ts (hunk 1 above)
Comment (banahaker) [reply src/router/index.ts]: 因為你們account 這個頁面的檔案名寫HomePage

Comment (SiongSng) [reply src/router/index.ts]: 沒有吧

**Review by SiongSng — APPROVED** [review]
LGTM

**Pull request merged**